app/experimenter/experiments/api/v5/types.py

```diff
@@ -302,6 +303,7 @@ def resolve_targeting_configs(root, info):
                 application_values=NimbusExperiment.TARGETING_CONFIGS[
                     choice.value
                 ].application_choice_names,
+                description=NimbusExperiment.TARGETING_CONFIGS[choice.value].description,
```

Comment:
Possible nit: is it normal to not put spaces around the = sign? I've seen this format in a few places but I'm not sure where it should be used 😅

Reply:
All the formatting is automatic, you don't have to worry about it ;)